keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -547,28 +585,40 @@ It's impossible to provide comprehensive guidance, but at the very high level
 These goals will help you determine what you need to measure (SLIs) in the next
 question. -->
 
+This feature should not affect the SLO of any components.
+
+The size of the OpenAPI scales as a cluster has more object types. In OpenAPI V2, aggregating the OpenAPI for all types was very resource intensive. In OpenAPI V3, aggregation is split and the aggregator will act as a proxy instead. Additional computation will be taken to build the OpenAPI V3 specs, and CRD specs will be aggregated after being built. We will monitor the current metrics to ensure that no regression is introduced with OpenAPI V3.
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 <!-- Pick one more of these and delete the rest. -->
 
-- [ ] Metrics
-  - Metric name:
-  - [Optional] Aggregation method:
-  - Components exposing the metric:
-- [ ] Other (treat as last resort)
-  - Details:
+A new metric will be added in CRD controller to measure the time taken to aggregate CRD OpenAPI specs.
+
+ - [X] Metrics
+  - Metric name: `crd_openapi_v3_aggregation_duration`
```

Comment:
@logicalhan is this a good name?

Reply:
Conventionally, we prefix the unit there as well, so in this case, it should be `crd_openapi_v3_aggregation_duration_seconds`

Reply:
Thanks Han! Updated.